### PR TITLE
feat(tokens): new Teradata branding

### PR DESCRIPTION
**Pull request by owilliams320**
## Description

Adding the new branding tokens based off Teradata's new branding tokens 🥇 

### What's included?
- Updating style dictionary tokens based off Material 3 using Teradatas new tokens
  - new tokens are now prefaced with the`theme` value
- including a new `teradata-theme` file in the `@covalent/core` package that includes references to new tokens. these variables can be used in conjuction with materials scss mixins for light/dark.
- Angular core: using appearance `outline` &  `Material Symbols Outlined` icon font as default values

### Web Components
- Exposing `forceOpen` property for the  `cv-app-shell` component
- Various import fixes for storybook stories
- Updates to storybook theming

#### Test Steps

- [ ] `npm run start`
- [ ] then open the local dev server a poke around to make sure everything is 🌞 
- [ ] finally this `npm run storybook` and ensure each component looks ok 👍🏽 

#### General Tests for Every PR

- [ ] `npm run start` still works.
- [ ] `npm run lint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build` still works.

##### Screenshots or link to StackBlitz/Plunker

**Storybook**
Screenshot 2024-02-07 at 8 41 08 PM
Screenshot 2024-02-07 at 8 41 17 PM

**Angular docs**
Screenshot 2024-02-07 at 8 43 28 PM
Screenshot 2024-02-07 at 8 43 57 PM

### CODE REVIEW

**Review by mikespudwebb — APPROVED** [review]
Looks really solid. I think I have more learning based questions than anything. :shipit:

**Review by fernando182gomez — APPROVED** [review]

File: apps/docs-app/src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <td-layout #layout [class]="activeTheme" [dir]="dir">
   <td-navigation-drawer
     sidenavTitle="Covalent"
-    [logo]="activeTheme === 'theme-dark' ? 'teradata' : 'teradata-dark'"
+    [logo]="activeTheme === 'theme-dark' ? 'teradata' : 'teradata'"
```
[the remark this answers is not in the document]
Comment (owilliams320) [reply apps/docs-app/src/app/app.component.html:4]: yup, so now the svg ive been using in the source code is transparent and using css variable `--cv-light-text-logo-on-background`  and `--cv-dark-text-logo-on-background` for the fill value.

File: apps/docs-app/src/assets/icons/teradata.svg
[the remark this answers is not in the document]
Comment (owilliams320) [reply apps/docs-app/src/assets/icons/teradata.svg:1]: hmm wierd comparison but probably in the code is removing the fill color. theres also some slight differences in the new logo

File: libs/angular-guided-tour/styles/guided-tour.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-$mat-icon-font: 'Material Icons', sans-serif;
+$mat-icon-font: 'Material Symbols Outlined', sans-serif;
```
[the remark this answers is not in the document]
Comment (owilliams320) [reply libs/angular-guided-tour/styles/guided-tour.scss:1]: can use same local version if we were pulling in from Covalent scss mixin the way the docs currently is doing

File: libs/angular/common/src/common.module.ts [hunk 3]
```diff
@@ -36,6 +37,16 @@ const TD_PIPES: Type<any>[] = [
...
 
+const TD_DEFAULT_ICON_OPTIONS = {
+  provide: MAT_ICON_DEFAULT_OPTIONS,
+  useValue: { fontSet: 'material-symbols-outlined' },
+};
+
+const TD_DEFAULT_FORM_FIELD_OPTIONS = {
+  provide: MAT_FORM_FIELD_DEFAULT_OPTIONS,
+  useValue: { appearance: 'outline' },
+};
```
[the remark this answers is not in the document]
Comment (owilliams320) [reply libs/angular/common/src/common.module.ts:48]: yes! was suggested by one of our designers awhile ago...kinda snuck this in, but it just makes so much sense

File: libs/components/src/app-shell/app-shell.stories.js [hunk 4]
```diff
@@ -80,9 +83,11 @@ const Template = ({ appName = '[App name]', sectionTitle = '' }) => {
         }
     }
     </style>
-    <cv-app-shell ${appName ? `appName="${appName}"` : ''} ${
-    sectionTitle ? `sectionName="${sectionTitle}"` : ''
-  }>
+    <cv-app-shell 
+    ${appName ? `appName="${appName}"` : ''} 
+    ${sectionTitle ? `sectionName="${sectionTitle}"` : ''}
+    ${forcedOpen ? `forcedOpen open` : ''}
```
[the remark this answers is not in the document]
Comment (owilliams320) [reply libs/components/src/app-shell/app-shell.stories.js:89]: so this is like a two parter to get the drawerer forced in its `open` state. Underneath most of our web components we have extend the material web components. The app-shell is extended from the [drawer](https://github.com/material-components/material-web/tree/mwc/packages/drawer). which is where the open property comes from

File: libs/components/src/textfield/textfield.ts [hunk 5]
```diff
@@ -18,6 +18,8 @@ export class CovalentTextField extends TextFieldBase {
       ${unsafeCSS(styles)}
     `,
   ];
+
+  override outlined = true;
```
[the remark this answers is not in the document]
Comment (owilliams320) [reply libs/components/src/textfield/textfield.ts:22]: this is for the web compents and the other change above is for the angular components.

File: libs/angular/common/styles/font/_font.scss [hunk 6]
```diff
@@ -1,41 +1,7 @@
...
-    /* Support for Safari and Chrome. */
-    text-rendering: optimizeLegibility;
-
-    /* Support for Firefox. */
-    -moz-osx-font-smoothing: grayscale;
-
-    /* Support for IE. */
-    font-feature-settings: 'liga';
-  }
+  @import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200&display=block');
```
[the remark this answers is not in the document]
Comment (owilliams320) [reply libs/angular/common/styles/font/_font.scss:4]: All the icons! with this we should get a large update of available icons to use https://fonts.google.com/icons

File: apps/docs-app/src/app/app.component.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikespudwebb) [reply apps/docs-app/src/app/app.component.html:4]: Ok. but those colors are in the SVG now? I'm not seeing how the word `teradata` here will be different when in light or dark mode.

[the remark this answers is not in the document]
Comment (owilliams320) [reply apps/docs-app/src/app/app.component.html]: totally missed that it was still a ternary. updated!

**Review by mikespudwebb — APPROVED** [review]
Looks great!!

**Pull request merged**